webclient/src/containers/Account/AddToBuddies.tsx

```diff
@@ -1,19 +1,16 @@
-// eslint-disable-next-line
-import React from "react";
-import { connect } from 'react-redux';
-import { Form, reduxForm } from 'redux-form'
+import React from 'react';
+import { Form } from 'react-final-form'
 
 import { InputAction } from 'components';
-import { FormKey } from 'types';
 
-const AddToBuddies = ({ handleSubmit }) => (
-  <Form onSubmit={handleSubmit}>
-    <InputAction action="Add" label="Add to Buddies" name="userName" />
+const AddToBuddies = ({ onSubmit }) => (
+  <Form onSubmit={values => onSubmit(values)}>
+    {({ handleSubmit }) => (
+      <form onSubmit={handleSubmit}>
+        <InputAction action="Add" label="Add to Buddies" name="userName" validate={() => true} />
```

Comment:
Should this be removed?
```suggestion
        <InputAction action="Add" label="Add to Buddies" name="userName" />
```